src/submission/validation-clinical/therapy.ts

```diff
@@ -169,13 +169,20 @@ const validateRadiationRecords = async (
     }
 
     // Therapy + Treatment are associated with Radiation
-    const currentTreatmentIsRadiation = treatment_type === 'radiation';
+    const currentTreatmentIsRadiation =
+      treatment_type &&
+      Array.isArray(treatment_type) &&
+      treatment_type.includes('Radiation therapy');
 
     const submittedTreatmentIsRadiation =
-      submissionTreatmentIdMatch && submissionTreatmentIdMatch.treatment_type === 'radiation';
+      submissionTreatmentIdMatch &&
+      Array.isArray(submissionTreatmentIdMatch.treatment_type) &&
+      submissionTreatmentIdMatch.treatment_type.includes('Radiation therapy');
```

Comment:
I think both of these can be shortened to remove the first undefined check, as `Array.isArray(undefined) // = false`

Reply:
Yes and replaced w/ `?` operators excellent

Reply:
Updated